### PR TITLE
Refactored data quality DAG

### PR DESCRIPTION
**Pull request by TJaniF**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,8 @@ To run the repository you will need to configure a Snowflake connection. With at
 - password: your Snowflake password
 - location: your Snowflake location (e.g. us-east-01)
 
+You will also need to provide the name of your database on line 14 of the `data_quality_checks_dag` and the name of your schema on line 1 in the checks.py file in the `/include` folder.
+
 The repository contains 3 dags in the dags folder. `create_table_1_dag` will create `table_1` in Snowflake, `create_table_2_dag` will create `table_2`. `create_table_2_dag` is triggered once `create_table_1_dag` has completed and `data_quality_checks_dag` is triggered once the both two dags have been completed (see: [Datasets and Data-Aware Scheduling in Airflow](https://www.astronomer.io/guides/airflow-datasets/)) and will run a set of data quality checks on both tables. All checks are set up to pass.
```
Comment (kentdanas) [inline README.md]: I would put this info into 3 bullet points, one for each DAG

File: README.md [hunk 2]
```diff
@@ -17,6 +17,8 @@ To run the repository you will need to configure a Snowflake connection. With at
 - password: your Snowflake password
 - location: your Snowflake location (e.g. us-east-01)
```
Comment (kentdanas) [inline README.md]: Isn't this field called "region"?

**Review by kentdanas — APPROVED** [review]
2 minor comments, otherwise looks good to me!

File: README.md (hunk 2 above)
Comment (TJaniF) [reply README.md]: Huh, yes it is. I have no idea how I came up with `location` here 😅

**Pull request merged**